plugins/modules/vmware_host_firewall_manager.py

```diff
@@ -60,6 +60,8 @@
     rules:
         - name: vvold
           enabled: True
+          allowed_hosts:
+            all_ip: True
```

Comment:
```suggestion
            all_ip: true
```
In YAML examples, it's better to use YAML `true`/`false` than Python `True`/`False` (same for existing samples, and the other new ones).

Reply:
I would prefer to keep this like that in this commit to be consistent with the rest of the module documentation.